core/src/main/java/org/kiwix/kiwixmobile/core/CoreApp.kt

```diff
@@ -78,15 +76,6 @@ abstract class CoreApp : Application() {
   }
 
   override fun onCreate() {
-    if (MissingSplitsManagerFactory.create(this).disableAppIfMissingRequiredSplits()) {
-      AlertDialog.Builder(this)
-        .setMessage(R.string.missing_split_version_for_objectbox)
-        .setPositiveButton(R.string.yes) { _, _ ->
-          android.os.Process.killProcess(android.os.Process.myPid())
-        }
-        .show()
-      return
-    }
```

Comment:
What'll happen if the user has an Unsupported architecture device? Should we re-open ticket #2904? Otherwise, it'll crash on unsupported devices.

Reply:
yes i have just reopen it.